docs/user/build-notifications.rst

```diff
@@ -17,8 +18,13 @@ Build Status Webhooks:
   Webhooks can be customized through your own template and a variety of variable substitutions.
 
 .. seealso::
-    :doc:`/guides/build-notifications`
-        All the practical steps for setting up build notifications
+    :doc:`/guides/build/email-notifications`
+        Enable email notifications on failed builds,
+        so you always know that your docs are deploying successfully.
+
+    :doc:`/guides/build/webhooks`
+        Steps for setting up build notifications via webhooks,
+        including examples for popular platforms like Slack and Discord.
 
     :doc:`pull-requests`
```

Comment:
I think we should note the limitations of the feature (eg. this doesn't work on PR builds) on this page as well, along with the howto. This page overall is not very useful, but it at least documents that we provide this feature, and show note it's higher level constraints.

Reply:
That's _noted_ now (as in: it has a `.. note::`)

I agree that the Feature description isn't very good... it's not "impressive" in the way it presents itself, and it's not very informative as a piece of "documentation reference".

As a general recipe, I think that a way we could improve some of these feature pages easily would be to add cross-reference to their API endpoints. IIRC, webhooks have an API.